### PR TITLE
save cash pickup details

### PR DESCRIPTION
**Pull request by Jordan-type**

### CODE REVIEW

File: routes/users.route.js [hunk 1]
```diff
@@ -286,14 +294,17 @@ router.post("/", async(req, res, next) => {
         msg += `CON Please enter your access pin to confirm to sell cUSD`
         res.send(msg)
     } else if (data[0] == '2' && data[1] == '3' && data[2] !== '' && data[3] !== '' && data[4] !== '' && data[5] !== '' && data[6] == '1' && data[7] !== '' && data[8] == null) {
+        number = 
         sellerMSISDN = phoneNumber
-        escrowMSISDN = phoneNumber // Todo: change to nigeria phone code '+234'
-        agentMSISDN = phoneNumber  // Todo: add agent phone number
+        escrowMSISDN = '+254712380880'  // Todo: change to nigeria phone code '+234' escrowMSISDN
+        agentMSISDN = '+254712380880'  // Todo: add agent phone number
```
Comment (Joda07) [inline routes/users.route.js:300]: Would this affect current production transaction. I don't see these values getting updated elsewhere by Users number

File: routes/users.route.js [hunk 2]
```diff
@@ -369,21 +384,37 @@ router.post("/", async(req, res, next) => {
...
         pickupPerson = data[6]
         recipientMSISDN = data[7]
         userInputPin = data[8]
 
         en_userInputPin = encryptionPin(userInputPin.toString())
         verificationId = generateVerificationId()
 
-        console.log('coinToSell', coinToSell, 'amountToSell', amountToSell, 'cashPickupLocation', cashPickupLocation, 'lga', lga, 'pickupPerson', pickupPerson, 'userInputPin', userInputPin, 'recipientMSISDN', recipientMSISDN)
+        // cashPickupLocation
+        if(data[4]==='1'){cashPickupLocation = 'Ikeja'}
```
Comment (Joda07) [inline routes/users.route.js:402]: 402 --> 417  what is  happening with the other location option.

**Review by Joda07 — COMMENTED** [review]
Please see review. Lets get clarification

File: routes/users.route.js (hunk 1 above)
Comment (Jordan-type) [reply routes/users.route.js:300]: for the escrow and agent phone number am waiting on PM  escrow should be the canza phone number and wallet address while the agent phone number is supposed to be a physical agent still waiting on her I add them manually on the database.

File: routes/users.route.js (hunk 2 above)
Comment (Jordan-type) [reply routes/users.route.js:402]: Let me check on my side  if the same is happening

Comment (Jordan-type) [reply routes/users.route.js:402]: the user is supposed to select the location they want to pick up their cash

**Pull request merged**